armi/utils/plotting.py

```diff
@@ -1224,27 +1223,27 @@ def _makeBlockPinPatches(block, cold):
     return patches, data, names
 
 
-def _makeComponentPatch(component, position, cold):
+def _makeComponentPatch(component, position, cold, cornersUp=False):
     """Makes a component shaped patch to later be used for making block diagrams.
 
     Parameters
     ----------
-        component: a component of a block
-
-        position: tuple
-            (x, y) position
-
-        cold: boolean
-            True if looking for dimension at cold temps
+    component: a component of a block
+    position: tuple
+        (x, y) position
+    cold: bool
+        True if looking for dimension at cold temps
+    cornersUp: bool, optional
+        If this is a HexBlock, is it corners-up or flats-up?
 
     Return
     ------
-        blockPatch: List
-            A list of Patch objects that together represent a component in the diagram.
+    blockPatch: list
+        A list of Patch objects that together represent a component in the diagram.
 
     Notes
     -----
-    Currently accepts components of shape DerivedShape, Helix, Circle, or Square
+    Currently accepts components of shape DerivedShape, Circle, Helix, Hexagon, or Square
```

Comment:
Not related to your PR, but this line says that the method accepts `DerivedShape`s. It does not appear to me that this is true.

Reply:
eone